topics/ecology/tutorials/MetaShARK_tutorial/tutorial.md

```diff
@@ -64,10 +64,10 @@ Finally, through the <b>MetaShARK R Shiny app</b> created by the french biodiver
 
 <br>
 
-> <comment-title>What is a **Data Paper**?</comment-title>
-> <p align="justify"> According to the [GBIF](https://www.gbif.org/data-papers) (Global Biodiversity Information Facility), 
+> <comment-title>What is a **Data Paper** ?</comment-title>
```

Comment:
the title itself doesn't process markdown yet due to som issues we had. Please use <b> (would issue suggestion but I'm on mobile)